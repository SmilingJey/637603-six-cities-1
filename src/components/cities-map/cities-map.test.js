import React from 'react';
import renderer from 'react-test-renderer';
import CitiesMap from './cities-map.jsx';

const mock = {
  places: [
    {
      id: 999,
      isPremium: true,
      picture: `test-image1.jpg`,
      price: 111,
      rating: 22,
      title: `Test title 1`,
      type: `Apartment`,
      coordinates: [52.3909553943508, 4.85309666406198],
    },
    {
      id: 222,
      isPremium: false,
      picture: `test-image2.jpg`,
      price: 222,
      rating: 100,
      title: `Test title 2`,
      type: `Room`,
      coordinates: [52.3909553943508, 4.85309666406198],
    },
  ],
  cities: [{
    name: `Amsterdam`,
    coordinates: [52.38333, 4.9],
  }],
};

it(`CitiesMap snapshot`, () => {
  const {places, cities} = mock;
  const tree = renderer.create(<CitiesMap places={places} city={cities[0]} />).toJSON();
  expect(tree).toMatchSnapshot();
});
