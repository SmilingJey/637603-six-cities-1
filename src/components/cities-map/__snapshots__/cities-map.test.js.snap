// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesMap snapshot 1`] = `
<section
  className="cities__map"
>
  <div
    id="map"
    style={
      Object {
        "height": "100%",
      }
    }
  >
    <div />
  </div>
</section>
`;
